include/triton/common/triton_json.h

```diff
@@ -243,6 +243,29 @@ class TritonJson {
       return TRITONJSON_STATUSSUCCESS;
     }
 
+    // Set/overwrite a string member, name, in this object with provided value
```

Comment:
Should we improve the documentation like below?

Set/overwrite a string member with provided name and value in this object ?

Reply:
updated.